storm-core/src/clj/backtype/storm/cluster.clj

```diff
@@ -90,9 +86,14 @@
   (update-storm! [this storm-id new-elems])
   (remove-storm-base! [this storm-id])
   (set-assignment! [this storm-id info])
-  ;adds nimbusinfo under /stormroot/code-distributor/storm-id
-  (setup-code-distributor! [this storm-id info])
+  ;; Sets up information related to key consisting of nimbus
```

Comment:
Sets --> sets? keep consistent with the comment style above, same for all below.